src/components/BasicSlider/BasicSlider.tsx

```diff
@@ -0,0 +1,50 @@
+import {useRef} from 'react';
+import Slider from '@mui/material/Slider';
+import {Box} from '@mui/material';
+import SecondaryTitle from '../typography/SecondaryTitle/SecondaryTitle';
+
+interface BasicSliderProps {
+  title: string;
+  ariaLabel?: string;
+  onChange: (values: number[]) => void;
+  value: number[];
+}
+
+const styles = {
```

Comment:
`styles` -> `sliderStyles` or you can create customized slider here for this case.